dhall-bash/dhall-bash.cabal

```diff
@@ -31,7 +31,7 @@ Library
         base                      >= 4.11.0.0 && < 5   ,
         bytestring                               < 0.11,
         containers                               < 0.7 ,
-        dhall                     >= 1.36.0   && < 1.37,
+        dhall                     >= 1.36.0   && < 1.38,
```

Comment:
If you want to make a new `dhall-bash` release, the `dhall-bash` version needs a bump too.

In principle, every package except `dhall` and `dhall-nixpkgs` could also be updated with a Hackage revision though.